### DISCUSSION TITLE
Tiger build going on 29 hours

### DISCUSSION THREAD

**Discussion by marlin-mixon** [mediagis/nominatim-docker#566] [Q&A] [answered]
I am trying to build a geocoder covering the state of Texas using Nominatim code (4,4). I got it to start building on a cloud server I use. I OK'd the option to include TIGER files for added accuracy. The main build took about nine hours on my cloud server. The Tiger files part of the build, so far, has taken an additional 29 hours with no apparent end in sight. I'm wondering how long it will take to complete. Anyone have experience with this? I'm beginning to wonder if the Tiger files might be for the full U.S.? I have approximately 10000 dots on the screen.  How many dots until completion?  At the beginning of this phase I received the following message:

 sudo -E -u nominatim nominatim add-data --tiger-data tiger-nominatim-preprocessed.csv.tar.gz
2024-06-16 22:42:38: Using project directory: /nominatim
2024-06-16 22:43:58: Found 3235 CSV files in tarfile with path tiger-nominatim-preprocessed.csv.tar.gz

Thanks for any help.

**Comment by mtmail** [comment, marked as answer]
It prints one dot for each 1000 street sections imported. Expect a totel of 32414 dots.

You can extract the archive, delete any filename not starting with '48' (or any file not containing `;TX;`) and recompress it into a new archive. That will cut the data by 90%.  [State FIPS (Federal Information Processing Standards) Codes ](https://www.icip.iastate.edu/maps/refmaps/STFIPS)

**Comment by marlin-mixon** [comment]
Thanks mtmail, that is so so useful and actionable!  So I am right at 48 hours and I'm a little over half way (16,784 dots).  If I stop now, cut the data down and restart it will complete in just under 10 hours at the current rate which beats 48 more hours.  Marlin
